File: Codechef/Reach_Fast/Reach_Fast.py
import math

T = int(input())
while T:
    T -= 1
    X, Y, K = map(int, input().split())
    if X <= Y:
        least = math.ceil((Y - X) / K)
    elif X >= Y:
        least = math.ceil((X - Y) / K)
    else:
        print(0)
    print(least)

File: Codechef/Reach_Fast/Reach_Fast.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int X, Y, K;
        float least;
        cin >> X >> Y >> K;
        if (X <= Y)
        {
            least = ceil((Y - X)/K);
        }
        else if (X >= Y)
        {
            least = ceil((X - Y)/K);
        }
        else
        {
            cout << 0 << endl;
        }

    }
    return 0;
}